p2p/utils.py

```diff
@@ -128,7 +130,7 @@ def request_to_curl(request):
         request.headers["Authorization"] = "Bearer P2P_API_KEY_REDACTED"
 
     # Format the headers
-    headers = ['"{0}: {1}"'.format(k, v) for k, v in request.headers.items()]
+    headers = ['"{0}: {1}"'.format(k, v) for k, v in list(request.headers.items())]
```

Comment:
Are all these list() calls necessary when we are iterating?

Reply:
They aren't in this case. This is the result of an automatic upgrade script.

Reply:
I've removed them from the `utils.py` file where they are unneeded